client/src/pages/trainer/courses/[id]/content/index.tsx

```diff
@@ -0,0 +1,57 @@
+import Breadcrumbs from '@/src/shared/components/Breadcrumbs';
+import Iframe from '@/src/shared/components/Iframe';
+import NavbarDetail from '@/src/shared/components/NavbarDetail';
+import SidebarContent from '@/src/shared/components/SidebarContent';
+import ContentLayout from '@/src/shared/layouts/ContentLayout/ContentLayout';
+import { Fragment } from 'react';
+
+const CourseContent: React.FC = () => {
+  const paths = [
+    {
+      text: 'Course',
+      url: '/'
+    },
+    {
+      text: 'Vue Introduction',
+      url: '/trainer/courses/1/content'
+    }
+  ];
+
+  return (
+    <Fragment>
+      <div className='ml-5 mt-5'>
+        <Breadcrumbs paths={paths}/>
+        <ContentLayout>
+          <div className='text-[20px] font-semibold my-5 text-black2'>Vue Introduction</div>
+          <div className='mb-5'>
+            <NavbarDetail />
+          </div>
+          <div className="flex">
+            <div>
+              <SidebarContent />
+            </div>
+            <div className='ml-7 flex-grow '>
+              <div className='text-[20px] font-semibold text-black2'>
+                Section 1
+              </div>
+              <div className='py-3'>
+                <Iframe
+                  src="https://www.youtube.com/embed/cJveiktaOSQ"
+                  className="border w-full"
+                />
+              </div>
+              <div className='text-[14px] font-semibold py-2'>
+                Description:
+              </div>
+              <div className='text-[14px] pb-24'>
+                Lorem ipsum dolor sit amet consectetur. Lectus sed interdum euismod rhoncus quis eu elementum. Sapien eu faucibus nisl tristique ultricies morbi pellentesque volutpat egestas.  Sapien eu faucibus nisl tristique ultricies morbi pellentesque volutpat egestas. Sapien eu faucibus nisl tristique ultricies morbi pellentesque volutpat egestas.
+              </div>
+            </div>
+          </div>
```

Comment:
This part of the code became very content related, like I mentioned before since content is just a tab of this page, tabs should not redirect user to another page like you do in your `<NavbarContent/>` component, 

Essentially there should be tabs that toggles the content for reference please check my implementation in this PR (https://github.com/framgia/sph-lms/pull/91/files#diff-d502af1fe583d69a1eb277857a54c28aa2e16985d7b68275ba80dbbf52a52ac8) 
Or you can just copy that Tabs folder in your branch and make use of it. its very much reusable, you might just need to update the style of it

The usage is very simple

```
<Tabs>
    <Tab title=''Content'>
         // first tab content goes here
    </Tab>
     <Tab title="Learners">
          // second tab content goes here
    </Tab>
    <Tab title="Settings">
          // third tab content goes here
     </Tab>
<Tabs>
```
For usage reference please check: https://github.com/framgia/sph-lms/pull/91/files#diff-dc92c71fbcf471db3c21d358a1a65bde6c5c7cb0d57c77b4cc585987e5ff0904
![image](https://github.com/framgia/sph-lms/assets/110364690/a929d552-4ca8-41ef-bd7f-e0960fdc60e1)

Please let me know if you have any questions about it